install/deploy_final_non-compute_node.md

```diff
@@ -138,7 +138,7 @@ The steps in this section load hand-off data before a later procedure reboots th
     The following commands create a `tar` archive of these files, storing it in a directory that will be backed up in the next step.
 
     ```bash
-    mkdir -pv "${PITDATA}"/prep/logs &&
+    PITBackupDateTime=$(date +%Y-%m-%d_%H-%M-%S); mkdir -pv "${PITDATA}"/prep/logs &&
```

Comment:
Since you don't really need this tar file to have its name be the same as the other one, it is probably better not to set this variable here. The further the distance between when a variable is set and when it is used, the greater the risk that the installer takes a break and ends up using a new shell.

Given that, I'd suggest just using the date command directly for this part, and setting the variable later, when it is needed.

```suggestion
    mkdir -pv "${PITDATA}"/prep/logs &&
```